paramParsers/tx.js

```diff
@@ -0,0 +1,195 @@
+const {
+  TxRaw, TxBody, AuthInfo,
+} = require('cosmjs-types/cosmos/tx/v1beta1/tx');
+const { pubkeyToAddress, pubkeyType: AminoPubKeyType } = require('@cosmjs/amino');
+const {
+  messageRegistryMap,
+  messageRegistry,
+} = require('./registry');
+const { bech32Prefix } = require('../config/config');
+
+function parseAny(any) {
+  const { typeUrl, value } = any;
+  const T = messageRegistryMap[typeUrl];
+  if (!T) {
+    // eslint-disable-next-line no-console
+    console.log(`Warning: cannot parse unregistered typeUrl ${typeUrl}`);
+    return {
+      parsed: false,
+      typeUrl,
+      result: any,
+    };
+  }
+  const decoded = messageRegistry.decode({ typeUrl, value });
+  const result = {
+    '@type': typeUrl,
+    ...decoded,
+  };
+  return {
+    parsed: true,
+    typeUrl,
+    result,
+  };
+}
+
+const messageHandler = {};
+
+function parseMsgs(msgs) {
+  const parsedMsgs = [];
+  for (const msg of msgs) {
+    const { parsed, typeUrl, result: parsedMessage } = parseAny(msg);
+    if (parsed) {
+      const handler = messageHandler[typeUrl];
+      if (handler) {
+        handler(parsedMessage);
```

Comment:
Not sure if it's better for `handler` to return parsed message, instead of just return void